admin/src/state/actions/Config.js

```diff
@@ -56,7 +56,7 @@ export function downloadZip(toggleNotification, formatMessage, post, get) {
   return async function(dispatch) {
     dispatch(setLoadingState(true));
     try {
-      const { message, base64Data, name } = await get('/config-sync/zip');
+      const { message, base64Data, name } = (await get('/config-sync/zip')).data;
```

Comment:
Good catch! Wasnt aware of this implementation, and couldn't test on Strapi v5. Thanks!